src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -159,9 +159,9 @@ public void ProcessStart_UseShellExecute_OnUnix_SuccessWhenProgramInstalled(bool
         [PlatformSpecific(~TestPlatforms.OSX)] // On OSX, ProcessName returns the script interpreter.
         public void ProcessNameMatchesScriptName()
         {
-            string scriptName = GetTestFileName();
+            const string scriptName = nameof(ProcessNameMatchesScriptName);
             string filename = Path.Combine(TestDirectory, scriptName);
-            File.WriteAllText(filename, $"#!/bin/sh\nsleep 600\n"); // sleep 10 min.
+            File.WriteAllText(filename, $"#!/bin/sh\n/bin/sleep 600\n"); // sleep 10 min.
```

Comment:
Is there a reason for changing the to explicitly using `/bin/sleep`? I don't think we care how the sleep is implemented.

Reply:
> I am not a Linux expert, but I guess that `sleep 600` could become a zombie right away only if there is a custom alias for the `sleep` command. I don't know if it's possible, but I think that we can just use full path to be 100% sure we are calling the actual `/bin/sleep`.

Reply:
I know on Alpine it's busybox, not an actual executable file, but it's symlinked to the same place so I guess that's OK:
```
LAPTOP-P6UJDVTA:~$ ls -l /bin/sleep
lrwxrwxrwx    1 root     root            12 Oct 21 09:23 /bin/sleep -> /bin/busybox
```